indico/modules/events/papers/settings.py

```diff
@@ -0,0 +1,26 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.modules.events.settings import EventSettingsProxy
+
+paper_reviewing_settings = EventSettingsProxy('paper_reviewing', {
+    'start_dt': None,
```

Comment:
I know, but IMO it'd still be cleaner to fix this here, no need to have a commit with known bugs that are fixed in a few lines ;)